app/controllers/sites/rss_feeds_controller.rb

```diff
@@ -1,108 +1,106 @@
 # frozen_string_literal: true
 
-module Sites
-  class RssFeedsController < Sites::SetupSiteController
-    include ActionView::Helpers::TextHelper
-    before_action :setup_rss_feed, only: [:show, :edit, :update]
-    before_action :setup_non_managed_rss_feed, only: [:destroy]
-
-    def index
-      @rss_feeds = @site.rss_feeds.order(:name)
-    end
+class Sites::RssFeedsController  < Sites::SetupSiteController
```

Comment:
- remove the extra space before the `<`
- disable the Style/ClassAndModuleChildren cop in `.rubocop.yml`